src/tuxedo-tomte

```diff
@@ -4102,10 +4134,8 @@ sub printLog {
 	if (!defined($type)) {
 		$type = 'l1';
 	}
-	else {
-		if ($type =~ /^[lt]+$/i) {
-			$type = "${type}1";
-		}
+	elsif ($type =~ /^[lt]+$/i) {
+		$type = "${type}1";
```

Comment:
Patch of "L/T/LT" (case insensitive) with forgotten logLevel in a shorter variant. (elsif instead else with following if)